### PR TITLE
ui: add rsx some options to gui

### PR DESCRIPTION
**Pull request by illusion0001**
Add the much needed disable hardware support for float 16 to advanced tab and 3D TV to debug

### CODE REVIEW

**Comment by illusion0001** [comment]
I might move 3d tv support to main GPU tab. Awaiting review from kd-11

**Comment by Megamouse** [comment]
these tooltips are not really informative

**Comment by illusion0001** [comment]
It's more of a placeholder for now. I'll rename it once kd suggest something better

**Comment by kd-11** [comment]
You can move 3D to main tab since it technically works. Tooltip for it is pretty straightforward, something like "Enables 3D stereo rendering. Note that only anaglyph viewing is supported at the moment".
As for float16, "Disables hardware half-float support which is known to cause problems in some rare cases on some GPUs." should be a good enough tooltip I think.

**Comment by illusion0001** [comment]
Pushed
image

File: rpcs3/rpcs3qt/tooltips.h [hunk 1]
```diff
@@ -93,7 +93,9 @@ class Tooltips : public QObject
 		const QString disable_fifo_reordering      = tr("Disables RSX FIFO optimizations completely. Draws are processed as they are received by the DMA puller.");
 		const QString gpu_texture_scaling          = tr("Force all texture transfer, scaling and conversion operations on the GPU.\nMay cause texture corruption in some cases.");
 		const QString strict_texture_flushing      = tr("Forces texture flushing even in situations where it is not necessary/correct. Known to cause visual artifacts, but useful for debugging certain texture cache issues.");
-
+		const QString disable_native_fp16          = tr("Disables hardware half-float support which is known to cause problems in some rare cases on some GPUs.");
+		const QString enable_3d                    = tr("Enables 3D stereo rendering.\nNote that only anaglyph viewing is supported at the moment.");
+
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/tooltips.h:98]: Why did this empty line change though?

Comment (illusion0001) [reply rpcs3/rpcs3qt/tooltips.h]: oops, now fixed

**Pull request merged**